README.md

```diff
@@ -1,69 +1,115 @@
+# MinIO Python Client SDK for Amazon S3 Compatible Cloud Storage [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io) [![Sourcegraph](https://sourcegraph.com/github.com/minio/minio-py/-/badge.svg)](https://sourcegraph.com/github.com/minio/minio-py?badge) [![Apache V2 License](https://img.shields.io/badge/license-Apache%20V2-blue.svg)](https://github.com/minio/minio-py/blob/master/LICENSE)
 
-MinIO Python SDK is Simple Storage Service (aka S3) client to perform bucket and object operations to any Amazon S3 compatible object storage service.
+The MinIO Python Client SDK provides straightforward APIs to access any Amazon S3 compatible object storage.
 
-For a complete list of APIs and examples, please take a look at the [Python Client API Reference](https://min.io/docs/minio/linux/developers/python/API.html)
+This Quickstart Guide covers how to install the MinIO client SDK, connect to MinIO, and create a sample file uploader.
+For a complete list of APIs and examples, see the [Python Client API Reference](https://min.io/docs/minio/linux/developers/python/API.html)
 
-## Minimum Requirements
-Python 3.7 or higher.
+These examples presume a working [3.7+ Python development environment](https://www.python.org/downloads/) and the [MinIO `mc` command line tool](https://min.io/docs/minio/linux/reference/minio-mc.html).
 
-## Download using pip
+## Install the Minio Python SDK
+
+### Using pip
 
 ```sh
 pip3 install minio
 ```
 
-## Download source
+### From GitHub
 
 ```sh
 git clone https://github.com/minio/minio-py
 cd minio-py
 python setup.py install
 ```
 
-## Quick Start Example - File Uploader
-This example program connects to an S3-compatible object storage server, make a bucket on that server, and upload a file to the bucket.
-
-You need the following items to connect to an S3-compatible object storage server:
+## Initialize a MinIO Client Object
 
-| Parameters | Description                                                |
-|------------|------------------------------------------------------------|
-| Endpoint   | URL to S3 service.                                         |
-| Access Key | Access key (aka user ID) of an account in the S3 service.  |
-| Secret Key | Secret key (aka password) of an account in the S3 service. |
+The MinIO client requires the following parameters to connect to an Amazon S3 compatible object storage:
 
-This example uses MinIO server playground [https://play.min.io](https://play.min.io). Feel free to use this service for test and development.
+| Parameter  | Description                                            |
+|------------|--------------------------------------------------------|
+| Endpoint   | URL to S3 service.                                     |
+| Access Key | Access key (user ID) of an account in the S3 service.  |
+| Secret Key | Secret key (password) of an account in the S3 service. |
```

Comment:
I would suggest that we would need to remove this and add comment in the example (the current example has that). BTW the parameter names are wrong.

Reply:
Corrected the table of args